tools_galaxyp.yaml

```diff
@@ -1,158 +1,925 @@
----
 install_repository_dependencies: true
 install_resolver_dependencies: true
 install_tool_dependencies: false
 
 tools:
-  - name: uniprotxml_downloader
+  - name: retrieve_ensembl_bed
     owner: galaxyp
     tool_panel_section_label: Get Data
 
-  - name: dbbuilder
+  - name: ms_wiff_loader
     owner: galaxyp
     tool_panel_section_label: Get Data
 
-  - name: unipept
+  - name: regex_find_replace
     owner: galaxyp
-    tool_panel_section_label: Get Data
+    tool_panel_section_label: Text Manipulation
+
+  - name: concatenate_multiple_datasets
+    owner: mvdbeek
+    tool_panel_section_label: Text Manipulation
+
+  - name: filter_by_substring_match
+    owner: alan-blakely
+    tool_panel_section_label: Filter and Sort
 
-  - name: map_peptides_to_bed
+  - name: column_join
+    owner: jjohnson
+    tool_panel_section_label: Join, Subtract and Group
+
+  - name: gffcompare_to_bed
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: idconvert
+  - name: bed_to_protein_map
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: idpquery
+  - name: translate_bed_sequences
     owner: galaxyp
     tool_panel_section_label: Convert Formats
 
-  - name: thermo_raw_file_converter
+  - name: peca
+    owner: caleb-easterly
+    tool_panel_section_label: Statistics
+
+  - name: limma_norm
+    owner: caleb-easterly
+    tool_panel_section_label: Statistics
+
+  - name: concat_fasta_files
```

Comment:
This tool is not maintained and also not needed, it just performs a `cat` operation. This can be done by the normal `cat` tools. The reason is that we do need to maintain the tools over time and for this, we would keep the tool number as much as possible.

Reply:
I see. Good catch.